### ISSUE TITLE
Multiple tabs open cause notifications of own messages

### ISSUE THREAD

**Issue by jancborchardt** [erming/shout#592]
When you have multiple tabs with Shout open, you get notifications for own messages you write. For example in a private conversation.

Also, you get notifications for the whois info which is displayed when starting a private conversation.

Both things should not happen by checking for that since it doesn’t make sense to send you notifications for that since you either just saw that / just sent that.

**Comment by YaManicKill** [comment]
This should probably be 2 issues, as the notifications of your own messages is a simple fix and obvious that its a bug (very simple fix as well). 

The whois info would probably need a discussion. I personally see no need for it, but it might be people prefer that.

**Comment by jancborchardt** [comment]
> I personally see no need for it, but it might be people prefer that.

Well, we need to simply make a proper design decision here instead of offering an essentially useless option to end users. ;) That’s why I was saying the notification should not be there.

**Comment by YaManicKill** [comment]
Yes, but the question is whether @erming had a good reason for allowing that in the first place, and whether or not we put it as a config. 

I would say just remove it. I don't see why we would need it.

**Comment by astorije** [comment]
The `whois` information is actually pretty nice to have when you open a private conversation. I myself use it from time to time and it's not like it's causing any big harm (just 4 messages). This can clearly be improved over time with a "User details" information paragraph somewhere, but my understanding is that the current solution was the cheapest way to do so (when it comes to something else than typing a command).

But let's keep this ticket about unnecessary notifications as this is a bug indeed.

I also noticed that when typing something, sending it and quickly changing tab, that message also appeared in a notification :-)

I'm flagging this as a quick and easy bug if someone wants to give it a shot. Thanks in advance!

**Comment by dgw** [comment]
> The `whois` information is actually pretty nice to have when you open a private conversation.
Yep, it just shouldn't create notifications. And that's why this ticket is being kept. :-)

**Comment by jancborchardt** [comment]
@astorije I also like the whois information – I wasn’t saying anything against that. Just the fact that there’s a notification for it when several tabs are open. ;) (As @dgw agrees it seems. ;)

**Comment by astorije** [comment]
Oops, sorry, I had misunderstood that! So many people in agreement on a same topic, this is rare and beautiful :-)

(By the way, @dgw, are you on the `#shout-irc` channel on freenode? Would be nice to have you there)
